### DISCUSSION TITLE
DTR/Server Info Bar

### DISCUSSION THREAD

**Discussion by goaaats** [goatcorp/Dalamud#745] [Announcements]
Plugins can now register to put stuff into the server info bar.

You get an object, you set the text and if it should be shown on it. You don't need to give users the option to turn this off in your plugin.
They can configure order and visibility in the Dalamud settings. That's it.

The options will only show up if the specific entry is registered.

The `DtrBar` class is registered as a plugin service. [See here for usage.](https://github.com/goatcorp/Dalamud/blob/master/Dalamud/Interface/Internal/Windows/DataWindow.cs#L1602-L1670)

![image](https://user-images.githubusercontent.com/16760685/151647028-87a6db28-ab22-4ac2-a9ae-c716e965212a.png)

**Comment by fixel99** [comment]
Is there a way to adjust spacing for this?

**Reply by goaaats** [reply]
It'll be an option sooner or later
